.github/workflows/ci.yml

```diff
@@ -101,46 +93,15 @@ jobs:
       - name: Install dependencies
         uses: ramsey/composer-install@v2
 
-      - name: Pull PHPStan cache
-        id: pull-phpstan-cache
-        uses: actions/cache/restore@v3
+      - name: Cache PHPStan
```

Comment:
Can you either update the PR title or only update the CS fixer workflow? Now the title doesn't match the changes in the PR.

Reply:
Done.